modules/odc-deploying-java-applications.adoc

```diff
@@ -26,7 +26,7 @@ Use the following instructions to upload a JAR file in the *Topology* view to de
 
 . In the *Name* field, enter a unique component name to name the associated resources.
 
-. In the *Resources* field, choose the resource type for your application.
+. Optional: You can select a different resource type from the default resource type if needed from the *Advanced options* -> *Resource type* drop-down list. 
 
 . In the *Advanced options*, click on *Create a Route to the Application* to configure a public URL for your deployed application.
```

Comment:
The previously stated generic suggestion of using the "To achieve an objective, perform an action" format applies here also. But I don't expect you to make sweeping changes to the whole document that's simply beyond the scope of the current PR. 

However, drop the "on" from this sentence to adhere to minimalism principles. For example:

In the *Advanced options*, click *Create a Route to the Application* to configure a public URL for your deployed application.